doc/api/next_api_changes/2019-04-26-AL.rst

```diff
@@ -0,0 +1,4 @@
+Deprecations
+````````````
+
+``matplotlib.checkdep_ps_distiller`` is deprecated.
```

Comment:
Why is it deprecated? Is it useless, or is there an alternative to use instead?

Reply:
It is useless and undocumented.
Well of course you *could* be trying to use this to figure out whether a ps distiller that happens to match the ones that matplotlib supports is installed.  In which case they can now just try to assign to `rcParams["ps.usedistiller"]` and see whether the setter complains.  But frankly I just don't think it's a realistic use case and adding this to the changelog is essentially noise, I would say.